sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/files/oems/packet

```diff
@@ -0,0 +1,3 @@
+/etc/systemd/system/packet-phone-home.service
```

Comment:
Do we want to add `/etc/systemd/system/multi-user.target.requires/coreos-metadata-sshkeys@core.service` here, too, because it's also in `ami`?

Reply:
same for DO and openstack